PresetParser/BuildingBlockProvider.cs

```diff
@@ -165,19 +211,33 @@ private bool ParseBuildingBlockerForAnno1800(XmlDocument ifoDocument, IBuildingI
         private bool ParseBuildingBlocker(XmlDocument ifoDocument, IBuildingInfo building, string variationFilename)
         {
             var variationFilenameWithoutExtension = Path.GetFileNameWithoutExtension(variationFilename);
-
+               
             try
             {
-                XmlNode node = ifoDocument.FirstChild[BUILDBLOCKER]?.FirstChild;
+                XmlNode node = ifoDocument.FirstChild[BUILDBLOCKER]?.FirstChild; 
+                building.BuildBlocker = new SerializableDictionary<int>();
```

Comment:
Is there a need for an empty `SerializableDictionary` when the result is `false`?
Before this change the result could be `false` and the `building.BuildBlocker` was `null`. There is a failing test because of this change.
I don't know if the test has to change or if the new logic is wrong.